app/controllers/coronavirus_form/check_answers_controller.rb

```diff
@@ -9,6 +11,9 @@ def show
   end
 
   def submit
+    validation_errors = validate_against_form_response_schema(session.to_h)
+    GovukError.notify(validation_errors) if validation_errors.any?
```

Comment:
This might want an exception or a string, I'm not sure what an object will do to it - it might work

https://github.com/alphagov/govuk_app_config/blob/master/lib/govuk_app_config/govuk_error.rb#L5

Reply:
I could `.to_s` the array?

Reply:
It would be nice to keep the structure... perhaps you could have that as the `extra: ...` part e.g. `GovukError.notify(FormResponseInvalid.new, extra: ...`), and provide a new error class? I'm guessing that would help Sentry group all of these together under one issue.

Reply:
I'll do that and fix up.